torchgen/dest/lazy_ir.py

```diff
@@ -129,6 +125,13 @@ def __call__(self, f: Union[NativeFunctionsGroup, NativeFunction]) -> List[str]:
     def lowering_function(self, f: Union[NativeFunctionsGroup, NativeFunction]) -> str:
         return ""
 
+    def can_be_reused_function(
+        self, f: Union[NativeFunctionsGroup, NativeFunction], node_ctor_args: str
+    ) -> str:
+        return f"""bool CanBeReused({node_ctor_args}) const {{
```

Comment:
@JackCaoG This is where I added a default method implementation. Let me know if it looks ok.

Reply:
this lgtm, thanks! Let me know if xla build still fail